test/augmentation/test_augmentation_mix.py

```diff
@@ -0,0 +1,194 @@
+from typing import Union, Tuple
+
+import torch
+from torch.testing import assert_allclose
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils  # test utils
+from kornia.augmentation import (
+    RandomMixUp,
+    RandomCutMix
+)
+
+
+class TestRandomMixUp:
+
+    def smoke_test(self, device):
+        f = RandomMixUp()
+        repr = "RandomMixUp(p=1.0, max_lambda=tensor(1.0), same_on_batch=False)"
+        assert str(f) == repr
+
+    def test_random_mixup_p1(self, device):
+        torch.manual_seed(0)
+        f = RandomMixUp()
+
+        input = torch.stack([torch.ones(1, 3, 4), torch.zeros(1, 3, 4)]).to(device)
+        label = torch.tensor([1, 0]).to(device)
```

Comment:
recommended way to initialize tensor is `torch.tensor([1, 0], device=device, dtype=dtype)` since it directly does initialization on the specified device and type.